data/simpy.gram

```diff
@@ -441,12 +442,12 @@ star_atom[expr_ty]:
 
 paren_ann_assign_target[expr_ty]:
     | '(' a=paren_ann_assign_target ')' { a }
-    | ann_assign_target
+    | non_simple_ann_assign_target
+    | a=NAME { set_expr_context(p, a, Store) }
 
-ann_assign_target[expr_ty]:
+non_simple_ann_assign_target[expr_ty]:
```

Comment:
I find a bit weird that this is called `non_simple_ann_assign_target` but there is not a `simple_ann_assign_target`. I know where the `non_simple` cames from, but I think is leaking the abstraction slightly.

Reply:
Maybe something like `ann_assign_target_subscript_attribute`?